emoji-copy@felipeftn/metadata.json

```
@@ -6,5 +6,5 @@
   "url": "https://github.com/felipeftn/emoji-copy",
   "settings-schema": "org.gnome.shell.extensions.emoji-copy",
   "gettext-domain": "emoji-copy",
-  "version": 21
+  "version": 22
```

The 23 version was already released! :sweat_smile:
Can you increase by one again to `23` pls? :pray: 
```suggestion
  "version": 23
```